### PR TITLE
Use evaluate-expression package

### LINKED ISSUE #66: evaluateExpression to package
Convert to github package in back-end repo.

### PR DESCRIPTION
**Pull request by CarlosNZ**
Fixes #66.

- Structured this repo folder so that `expression-evaluator` is its own "package", within modules folder. Can be worked on and developed as part of this repo, but can be published as its own package
- published as Github package within this repo: https://github.com/openmsupply/application-manager-server/packages/433685
- changed `.ts` files to use package rather than importing from modules folder
- Added registry details to `.npmrc` and `.yarnrc` so that package can be imported from Github npm registry.

You'll need to authenticate once with Github npm registry in order to download the package.

From [Github docs](https://docs.github.com/en/free-pro-team@latest/packages/using-github-packages-with-your-projects-ecosystem/configuring-npm-for-use-with-github-packages):

> To authenticate by logging in to npm, use the `npm login --registry=https://npm.pkg.github.com` command, replacing USERNAME with your GitHub username, TOKEN with your personal access token, and PUBLIC-EMAIL-ADDRESS with your email address.
> 
> ```
> $ npm login --registry=https://npm.pkg.github.com
> Username: USERNAME
> Password: TOKEN
> Email: PUBLIC-EMAIL-ADDRESS
> ```

See [here](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token) for how to generate your personal access token.

Basically, it should work exactly the same as a normal npm package except it's hosted on Github's npm registry rather than npmjs.

Will import package into front-end repo.

### CODE REVIEW

**Review by nmadruga — APPROVED** [review]

**Comment by nmadruga** [comment]
@CarlosNZ Sorry, I forgot to say here - can you add a document (can be straight on the master) explaining what is required for using the package please? Otherwise we have to find this again to fix this problem:
Screen Shot 2020-10-15 at 10 29 27 AM

* Also that link doesn't say, but that's how I had to have my token generate to `read package` to give to the npm registry:
Screen Shot 2020-10-15 at 10 12 37 AM

**Comment by CarlosNZ** [comment]
Yeah sure thing.

**Comment by CarlosNZ** [comment]
Documentation done: https://github.com/openmsupply/application-manager-server/wiki/Query-Syntax
